src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
Please reintegrate this. I really don't see any advantage to have this outsourced and replicated for every plugin.
We should provide this out-of-the-box.